view/view.go

```diff
@@ -50,29 +49,44 @@ func getDefaultConfig() goview.Config {
 		Master:    "layouts/default",
 		Partials:  getPartialList(),
 		Funcs: template.FuncMap{
-			"assetsCSS":  helpers.AssetsCSS,
-			"isActive":   helpers.IsActive,
-			"renderFile": helpers.RenderFile,
-			"renderIcon": helpers.RenderIcon,
-			"urlFor":     helpers.UrlFor,
-			"toKebab":    helpers.ToKebabCase,
-			"toTitle":    helpers.ToTitleCase,
+			"assetsCSS":      helpers.AssetsCSS,
+			"isActive":       helpers.IsActive,
+			"renderFile":     helpers.RenderFile,
+			"renderIcon":     helpers.RenderIcon,
+			"urlFor":         helpers.UrlFor,
+			"urlForID":       helpers.UrlForID,
+			"toKebab":        helpers.ToKebabCase,
+			"toTitle":        helpers.ToTitleCase,
+			"formatDateTime": helpers.FormatDateTime,
 		},
 	}
 }
 
 func getPartialList() []string {
 	partials := []string{}
-	files, err := ioutil.ReadDir(PARTIAL_PATH)
+
+	err := filepath.Walk(ROOT_VIEW_PATH, func(path string, info fs.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if !info.IsDir() {
+			fileName := info.Name()
+			fileExtension := filepath.Ext(fileName)
+
+			if fileExtension == ".html" && strings.HasPrefix(fileName, "_") {
+				filePath := strings.Split(path, ROOT_VIEW_PATH)[1]
+				filePathWithoutExt := filePath[:len(filePath)-len(fileExtension)]
+				partials = append(partials, filePathWithoutExt)
+			}
+		}
```

Comment:
Instead of registering only HTML files under the `partials` directory, list all the HTML files with file name prefixed with `_` and register them so we can split UI part into partials and place them in the same directory

Reply:
That's very nice. Do you think should we submit and add this to the goview repository as well? 😄

Reply:
Good idea! I added [an issue on goview repository](https://github.com/foolin/goview/issues/32) 😄